typescript/vrotest/src/build.ts

```diff
@@ -63,6 +49,16 @@ export default async function (flags: BuildCommandFlags) {
 	const modules: Record<string, ModuleDescriptor> = {};
 	const configurations: Record<string, ConfigCategory> = {};
 	const resources: Record<string, ResourceCategory> = {};
+    const customTestsConfigPath = path.join(flags.projectRoot, constants.TEST_CONFIG_PATH);
+    const hasCustomTestsConfig = fs.pathExistsSync(customTestsConfigPath);
+    const setupFramework = hasCustomTestsConfig
+        ? undefined
+        : (
+            flags.testFrameworkPackage === "jest"
```

Comment:
I was wondering whether it is worth converting this logic to a switch case and putting it in a function so that it is easier to extend down the line if we introduce more frameworks.
that being said, I don't think that we will introduce another test framework soon

Reply:
Or even better we could convert that to a factory. I was thinking of changing this to a factory either way so I could definitely do that, I didn't because I can't see introducing support for another framework as well and don't want to add more complexity to the code if not necessary... So I would appreciate your point of view as well and we can apply changes as we decide collectively :)